content/language/r/develop.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Use containers for R development
+keywords: R, local, development
+description: Learn how to develop your R application locally.
+---
+
+## Prerequisites
+
+Complete [Containerize a R application](containerize.md).
+
+## Overview
+
+In this section, you'll learn how to set up a development environment for your containerized application. This includes:
+
+- Adding a local database and persisting data
+- Configuring Compose to automatically update your running Compose services as you edit and save your code
+
+## Get the sample application
+
+You'll need to clone a new repository to get a sample application that includes logic to connect to the database.
+
+Change to a directory where you want to clone the repository and run the following command.
+
+```console
+$ git clone https://github.com/mfranzon/r-docker-dev.git
+```
```

Comment:
Can we use the same repository throughout the guide? So users don't have to clone two different ones. The `r-docker-dev` repository could be used from the beginning IMO, just comment out the compose parts and delete the db secret.

Reply:
You are right! I have add more information about the `src_db/` folder